tasks/zadanie_7.php

```diff
@@ -14,4 +14,12 @@
 
 // ...
 
-echo "Klient A wydał więcej";
\ No newline at end of file
+echo "Klient A wydał więcej";
+
+if ($customerA > $customerB) {
+    echo "Klient A wydał więcej";
+} elseif ($customerA == $customerB) {
+    echo "Klient A wydał tyle samo co klient B";
+} else ($customerA < $customerB) {
```

Comment:
w sytuacji jak mamy else - to już nie dajemy warunku ;)
"else" możemy traktować jak powiedzenie "w każdym innym wypadku"

```suggestion
} else {
```